crates/miden-lib/asm/kernels/transaction/lib/memory.masm

```diff
@@ -113,17 +113,17 @@ const.BLOCK_METADATA_PTR=828
 # The memory address at which the note root is stored
 const.NOTE_ROOT_PTR=832
 
-# CHAIN MMR
+# PARTIAL BLOCKCHAIN
 # -------------------------------------------------------------------------------------------------
 
 # The memory address at which the chain data section begins
-const.CHAIN_MMR_PTR=1200
+const.PARTIAL_BLOCK_CHAIN_PTR=1200
```

Comment:
Similar to to the other PR, I would spell "blockchain" as one word - i.e., `PARTIAL_BLOCKCHAIN_PTR`.